language/travis.js

```diff
@@ -0,0 +1,7 @@
+const shell = require('shelljs')
+
+if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST === 'false') {
+  const zanataUpload = shell.exec('yarn zanata:upload', { silent: true }).stdout
```

Comment:
What does the stdout look like?  I want to make sure nothing secret is exposed.

Reply:
<img width="717" alt="zanata" src="https://user-images.githubusercontent.com/230549/31094013-d81bde7c-a770-11e7-9a16-44a5908a515e.png">
Here is what the std out looks like